src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -162,14 +160,6 @@ internal static string GitCommitId
             }
         }
 
-        internal static Version CLRVersion
```

Comment:
What is our stance on removing FullCLR compatibility of the code? If we are dropping that requirement and only focusing on CoreCLR, then you should remove PSCLRVersionName from line 77 as well.